unicorecmsmnmcameroon/templates/home.pt

```diff
@@ -22,22 +22,38 @@
             </div>
             <div tal:repeat="page view.get_featured_category_pages(cat.uuid)"
                  tal:attributes="{'class':'listing'} if page.image_host and page.image else {'class': 'listing no-pic'}">
+                <div class="h2"><a href="/content/detail/${page.uuid}/">${page.title}</a></div>
                 <condition tal:omit-tag="" tal:condition="page.image_host and page.image">
-                    <img alt="${page.title}" src="${view.get_image_url(page.image_host, page.image, 45, 45)}"/>
+                    <img alt="${page.title}" src="${view.get_image_url(page.image_host, page.image, 90, 90)}"/>
                 </condition>
-                <div class="h2"><a href="/content/detail/${page.uuid}/">${page.title}</a></div>
                 <p>${page.description}</p>
                 <div class="clr"></div>
             </div>
 
             <div class="clr"></div>
             <div class="pagination more category-${cat.slug}" tal:condition="len(view.get_featured_category_pages(cat.uuid)) > 1">
-                <a href="/content/list/${cat.uuid}/"><span i18n:translate="More">More</span> &gt;</a>
+                <a href="/content/list/${cat.uuid}/"><span i18n:translate="More">Read more posts</span></a>
             </div>
         </div>
 
-        <div class="declaration">
-            <div class="pagination"><a href="/contact/" class="contact" i18n:translate="Contact">Contact</a> | <a href="/credits/" class="terms" i18n:translate="Credits">Credits</a> | <a href="/about/" class="terms" i18n:translate="About">About</a></div>
-        </div><!--/declaration-->
+        <div class="articles">
+            <div class="h1">Connect With Us</div>
+            <div class="links">
+                <a href="http://mnm-cm-185.apollo.unicore.io"><img src="/static/img/facebook-logo.png">Facebook</a>
```

Comment:
I did check this with her, she requested that the facebook links point to the i.org sites